doc/members/adding_member.rst

```diff
@@ -3,11 +3,13 @@ Adding New Members
 
 It is possible for existing members to add new members to the consortium after a CCF network has been started.
 
-.. note:: The maximum number of active consortium members at any given time is 255.
+.. note:: The maximum number of active recovery members (i.e. those with a recovery share) at any given time is 255.
```

Comment:
you probably want to say something like "The maximum number of allowed active ...."